src/Libraries/CoreNodes/Color.cs

```diff
@@ -29,7 +29,7 @@ public byte Red
         /// <summary>
         ///     Find the green component of a color, 0 to 255.
         /// </summary>
-        /// <returns name="green">int between 0 and 255 inclusive.</returns>
+        /// <returns name="integer">Green value for RGB color model, int between 0 and 255 inclusive.</returns>
```

Comment:
`int`

Reply:
you can say "integer" in the description.

Reply:
For this one I saw we sometimes use `int` sometimes use `integer` in code base, do we want to unify that?

Reply:
We also use the term `number` and `double` interchangeably. I'm in favour of the more verbose calls here towards `integer` over int for overall clarity.

Reply:
I prefer int, double, bool, and string like it's in most languages but that's just me.

Reply:
I have set up a poll here (https://forum.dynamobim.com/t/request-for-feedback-language-preferences/55066) to gauge the interest of the Dynamo Community in this.

Reply:
@martinstacey - Poll ran to a conclusion. Please follow pattern explained inside of Slack channel. Copied here for historical record: 

After internal communication and a public poll on the Dynamo Forums: https://forum.dynamobim.com/t/request-for-feedback-language-preferences/55066, can we please run with the following amendments to all submitted nodes:

- Please use short-form terms for all input port names for only commonly used DesignScript base type abbreviations, and long-form terms in the hover description. Short form terms taken from  https://dynamobim.org/wp-content/links/DesignScriptGuide.pdf
- This means things such as **func** or **comb** `do` become long-form. But things such as **bool** or **int** stay short-form.

Reply:
addressed in 8252853